File: app/src/main/java/com/admin/theapp/model/FirebaseHotelModel.java
package com.admin.theapp.model;

import com.google.firebase.database.IgnoreExtraProperties;

@IgnoreExtraProperties
public class FirebaseHotelModel {
    private long   id;
    private String name;
    private String address;
    private double stars;
    private double distance;
    private String suites_availability;
    private double lat;
    private double lon;
    private String image;

    public FirebaseHotelModel() {
    }

    public FirebaseHotelModel(long id, String name, String address, double stars, double distance, String suites_availability, double lat, double lon, String image) {
        this.id = id;
        this.name = name;
        this.address = address;
        this.stars = stars;
        this.distance = distance;
        this.suites_availability = suites_availability;
        this.lat = lat;
        this.lon = lon;
        this.image = image;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getAddress() {
        return address;
    }

    public String getSuites_availability() {
        return suites_availability;
    }

    public double getStars() {
        return stars;
    }

    public double getDistance() {
        return distance;
    }

    public double getLat() {
        return lat;
    }

    public double getLon() {
        return lon;
    }

    public String getImage() {
        return image;
    }
}

File: storage/src/main/java/com/admin/theapp/firebase/model/FirebaseHotelModel.kt
package com.admin.theapp.firebase.model

import com.google.firebase.database.IgnoreExtraProperties

@IgnoreExtraProperties
class FirebaseHotelModel {
    val id: Long
    val name: String
    val address: String
    val stars: Double
    val distance: Double
    val suites_availability: String
    val lat: Double
    val lon: Double
    val image: String

    constructor() {
        id = 0
        name = ""
        address = ""
        stars = 0.0
        distance = 0.0
        suites_availability = ""
        lat = 0.0
        lon = 0.0
        image = ""
    }

    constructor(id: Long, name: String, address: String, stars: Double, distance: Double, suites_availability: String, lat: Double, lon: Double, image: String) {
        this.id = id
        this.name = name
        this.address = address
        this.stars = stars
        this.distance = distance
        this.suites_availability = suites_availability
        this.lat = lat
        this.lon = lon
        this.image = image
    }
}
